.github/workflows/acceptance.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
         with:
           config: test/kind/config.yaml
           node_image: kindest/node:v${{ matrix.kind-k8s-version }}
+          version: v0.14.0
 
       # Skip CSI tests if K8s version < 1.16.x
```

Comment:
If you want, we could also remove the < 1.16.x check step here (and the `check_skip_csi` function and calls in test/acceptance/csi.bats.

Reply:
:+1: